src/query.rs

```diff
@@ -1,30 +1,32 @@
-use crate::de::wbem_class_de::from_wbem_class_obj;
-use crate::result_enumerator::{IWbemClassWrapper, QueryResultEnumerator};
-use crate::BStr;
 use crate::{
-    connection::WMIConnection, de::meta::struct_name_and_fields, utils::check_hres, WMIError,
+    result_enumerator::{IWbemClassWrapper, QueryResultEnumerator},
+    de::wbem_class_de::from_wbem_class_obj,
+    de::meta::struct_name_and_fields,
+    utils::check_hres,
+    connection::WMIConnection,
+    BStr,
+    WMIError, WMIResult,
 };
-use log::trace;
-use serde::de;
-use std::collections::HashMap;
-use std::ptr;
-use std::ptr::NonNull;
-use winapi::um::wbemcli::IWbemClassObject;
 use winapi::{
-    shared::ntdef::NULL,
-    um::{
-        wbemcli::IEnumWbemClassObject,
-        wbemcli::{
-            WBEM_FLAG_FORWARD_ONLY, WBEM_FLAG_RETURN_IMMEDIATELY, WBEM_FLAG_RETURN_WBEM_COMPLETE,
-        },
+    um::wbemcli::{
+        IWbemClassObject,
+        IEnumWbemClassObject,
+        WBEM_FLAG_FORWARD_ONLY,
+        WBEM_FLAG_RETURN_IMMEDIATELY,
+        WBEM_FLAG_RETURN_WBEM_COMPLETE,
     },
+    shared::ntdef::NULL,
 };
+use std::{ptr::{self, NonNull}, collections::HashMap, time::Duration};
+use log::trace;
+use serde::de;
 
 pub enum FilterValue {
     Bool(bool),
     Number(i64),
     Str(&'static str),
     String(String),
+    Is_A(&'static str),
```

Comment:
I think this should be `IsA`, and the function `is_a()` (just the expected Rust conventions)

Reply:
I should have done that from the beginning, but for some reason, I focused on `IsA`.
Anyway, I've changed this now.